app/services/usuario_service.py

```
@@ -0,0 +1,280 @@
+    def crear_nuevo_usuario(self, form, current_user):
+        """
+        Crea un nuevo usuario.
+        Devuelve el usuario creado o None si hay error.
+        """
+        try:
+            # Validaciones de existencia (username, cédula, email)
+            existing_user_username = Usuario.query.filter_by(username=form.username.data).first()
+            existing_user_cedula = Usuario.query.filter_by(cedula=form.cedula.data).first()
+            existing_user_email = None
+            if form.email.data:
+                 existing_user_email = Usuario.query.filter_by(email=form.email.data).first()
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)